src/cpp/rtps/reader/StatelessReader.cpp

```diff
@@ -238,12 +238,15 @@ bool StatelessReader::matched_writer_is_matched(
 bool StatelessReader::change_received(
         CacheChange_t* change)
 {
-    // Only make visible the change if there is not other with bigger sequence number.
+    // std::cout << "[FAST STATELESS READER] change_received" << std::endl;
```

Comment:
Seems this was left here unintentionally

Reply:
:+1: